demos/src/character_control_systems/platformer_control_systems.rs

```diff
@@ -677,33 +673,65 @@ impl UiTunable for FallingThroughControlScheme {
     }
 }
 
+/// A simple component to control the camera use by the `shooter_like` and `platformer_3d` demos.
+/// In the `shooter_like` demo, this component is mutated in by the player movement.
+/// In the `platformer_3d` demo, the camera position is updated via the UI (requires the "egui" feature).
+/// This component is not used in the `platformer_2d` demo.
 #[derive(Component)]
-pub struct ForwardFromCamera {
-    pub forward: Vector3,
-    pub pitch_angle: Float,
+pub enum CameraController {
+    /// A camera controller that follows the player
+    Following {
```

Comment:
A bit of a bikeshed, but I don't like these names. When I hear a camera described as "following" what I have in mind is these floating third person cameras from N64 platformers that follow the player around.